nixos/doc/manual/development/settings-options.section.md

```diff
@@ -146,6 +146,27 @@ have a predefined type and string generator already declared under
     :   Outputs the given attribute set as an Elixir map, instead of the
         default Elixir keyword list
 
+`pkgs.formats.php { finalVariable }` []{#pkgs-formats-php}
+
+:   A function taking an attribute set with values
+
+    `finalVariable`
+
+    :   The variable that will store generated expression (usually `$config`). If set to `null`, generated expression will contain `return`.
```

Comment:
It's a bit confusing on whether the `$` should be part of it. Consider either cleaning up the docs, or removing the need to specify the `$`